docs/genai.md

```diff
@@ -0,0 +1,46 @@
+# Endpoints for Generative Use Cases {#ovms_docs_genai}
```

Comment:
This title is descriptive, but from the menu point of view it's quite long (at the first sight you don't see what it means).
Maybe we could shorten it and start with `GenAI` phrase so it catches the eye of the viewer like:
`GenAI Endpoints` or `GenAI Use Cases` ?